### ISSUE TITLE
Make a public version of the milestone internal document

### ISSUE THREAD

**Issue by fharper** [meilisearch/devrel#246]
Make a public version of the [milestone document](https://www.notion.so/meilisearch/Milestones-1a3d9358295c436d8df6689e0444dc1c) (private) in English. Add a section at https://github.com/meilisearch/devrel/blob/main/assets/README.md 

Ideally, I would need it before the EOW, so we can share with the company which will do our rebranding.

**Comment by tpayet** [comment]
I didnt translate everything yet but I sorted what could be public and what not ^^ https://www.notion.so/meilisearch/Publishable-milestones-f4b8ce4b18974baaa846576fae5d8a5a
I wonder how we will maintain this as I suspect curqui will update only the private one in Notion for now ^^

**Comment by fharper** [comment]
I need to discuss with @curquiza but as you all know, I don't like duplicating content so I would suggest a hybrid approach where everything we can share goes in this markdown, and the real private stuff (like when you give me one million euros to approve this PR) will go in Notion, what do you think?

**Comment by curquiza** [comment]
Yes, I don't like duplicated content either. However I started the private document for fun, I don't want this to become another responsibility for me just because I started to do fun stuff for Meili 🙂
I already know if it's too complicated for me (thinking every time "is it sharable or not?" + having two places to fill the information) I'll naturally stop doing it 😂
I will keep doing it the one I fill because I like it, but we definitely need someone else to fill the public part 🙂

**Comment by fharper** [comment]
Is there a lot of things that are private? As in do we need a private version?

**Comment by tpayet** [comment]
Not that much, since @curquiza started it for fun and she don't want to get this new responsibility if we do make this public we would need someone else to take ownership for the public version ^^

**Comment by fharper** [comment]
I think it should be a company responsibility, like everyone can add items in it by doing a PR. WDYT?

**Comment by curquiza** [comment]
I'm afraid "everyone's responsibility" will become "no one responsibility". A lot of teams have this issue when requesting a team instead of a unique person on a PR for example. But maybe it's the best solution to start.

**Comment by fharper** [comment]
@curquiza is it that more complicated for you to just do a PR for new things that can be public and forget the rest? I would assume we don't have news every week?

**Comment by fharper** [comment]
P1 since we will need that information for the rebranding

**Comment by fharper** [comment]
@CaroFG are you comfortable enough in English to do the translation? In any cases, Tommy would review them before we merge.

**Comment by CaroFG** [comment]
I can do it

**Comment by fharper** [comment]
Perfect, thanks!

**Comment by ferdi05** [comment]
I love this idea and @curquiza too. I think that would be great content for the new website.
Something I can work on with our new developer advocate, it's a funny way to tell the story of the company (additionally to what is done by the founder's team during the onboarding of new team members).

@qdequele wdyt?

**Comment by qdequele** [comment]
@ferdi05 could be great to have it next to the about us page in the landing, I agree :)

**Comment by ferdi05** [comment]
I should have given an update on this earlier.
A while ago, @davelarkan helped translate this document.
Since then, @laurenemeili said she would be interested in using it, and I sent her the document. I don't really know where things are at the moment, this could be worth considering adding to the landing. I will try to send this issue to the related repo.

**Comment by shivaylamba** [comment]
This was an amazing activity!
